docs/source/conventions.rst

```diff
@@ -3,6 +3,16 @@ Common Conventions
 
 This document contains common conventions that SimpleITK filters, and objects follow. It is intended to describe the interfaces that users should use and developers should implement. If a method or class does not specify different behavior or default values then those described here, it should be assumed that it follows the following conventions.
 
+Naming
+......
+
+The object oriented interface of process objects are named based on the specific operation they perform on their input image(s)
+with a suffix of **ImageFilter**, the ITK convention. For example, `AbsImageFilter <https://simpleitk.org/doxygen/latest/html/classitk_1_1simple_1_1AbsImageFilter.html>`_, `BinaryMinMaxCurvatureFlowImageFilter <https://simpleitk.org/doxygen/latest/html/classitk_1_1simple_1_1BinaryMinMaxCurvatureFlowImageFilter.html>`_ etc. For almost all process objects there is also a procedural interface. One or more functions which
+allow you to set parameter values and invoke the operation in one call. This results in concise code and is useful if you only need to
```

Comment:
^you^a developer^

I'm not a fan of "you" unless it's really called for.

Reply:
Removed, though was initially considering changing it to ewe, even sheep can use the procedural interface 😉 .